gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
+#Tue Feb 15 15:44:28 IST 2022
 distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionSha256Sum=8cc27038d5dbd815759851ba53e70cf62e481b87494cc97cfd97982ada5ba634
```

Comment:
Gradle checksums are required to speed up the process of verifying Gradle for F-Droid